django/views/decorators/csrf.py

```diff
@@ -51,9 +53,17 @@ def csrf_exempt(view_func):
 
     # view_func.csrf_exempt = True would also work, but decorators are nicer
     # if they don't have side effects, so return a new function.
-    @wraps(view_func)
-    def wrapper_view(*args, **kwargs):
-        return view_func(*args, **kwargs)
 
-    wrapper_view.csrf_exempt = True
-    return wrapper_view
+    if iscoroutinefunction(view_func):
+
+        async def _view_wrapper(request, *args, **kwargs):
+            return await view_func(request, *args, **kwargs)
+
+    else:
+
+        def _view_wrapper(request, *args, **kwargs):
+            return view_func(request, *args, **kwargs)
```

Comment:
This commit mirrors the logic of the [commit](https://github.com/django/django/pull/16874/commits/23abec9192c88f9372e5784ae69c9aa5be2b21be) which makes the `@no_append_slash` decorator able to handle asynchronous views. That commit also added the initial `request` argument (rather than just having `*args` and `**kwargs`, so I assume it's fine for this commit to do the same?